SwiftUIDZ/SwiftUIDZ/TabBarView/TabBarView.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  TabBarView.swift
+//  SwiftUIDZ
+//
+//  Created by Алена Панченко on 04.02.2023.
+//
+
+import SwiftUI
+
+/// Таб бар приложения
+struct TabBarView: View {
+    
+    // MARK: - Public Properties
+    
+    var body: some View {
+        ZStack {
+            GradientNavigationView()
+            VStack {
+                TabView {
+                    SelectFurnitureView()
+                        .tabItem {
+                            Label("", systemImage: "window.awning")
```

Comment:
```suggestion
                            Label("", systemImage: "window.awning")
```
Вынеси в константы. И далее во всех tabItems..